.github/automation/x64/build_linters.sh

```diff
@@ -21,7 +21,9 @@ if [[ "$ONEDNN_ACTION" == "configure" ]]; then
           -DDNNL_CPU_RUNTIME=OMP \
           -DDNNL_GPU_RUNTIME=OCL \
           -DDNNL_WERROR=ON \
-          -DDNNL_BUILD_FOR_CI=ON
+          -DDNNL_BUILD_FOR_CI=ON \
+          -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
+      for file in `git diff --name-only "$1" | grep -E '\.cpp$'`; do clang-tidy --header-filter='' ../$file; done
```

Comment:
This filter will not catch regressions in header files (once header file checks are actually enabled).